onnxruntime/python/tools/transformers/shape_infer_helper.py

```diff
@@ -42,6 +42,7 @@ def infer(self, dynamic_axis_mapping):
     # override _preprocess() to avoid unnecessary model copy since ctor copies the model
     def _preprocess(self, in_mp):
         self.out_mp_ = in_mp
+        self.graph_inputs_ = dict([(i.name, i) for i in list(self.out_mp_.graph.input)])
```

Comment:
Is graph_inputs_ used?

Reply:
yes, https://github.com/microsoft/onnxruntime/blob/225439193e484e724d96f89c19da2f3d8794e9c6/onnxruntime/python/tools/symbolic_shape_infer.py#L401